Translations/French.csv

```diff
@@ -1342,7 +1342,7 @@ go.to.shuttle,Se déplacer vers la navette,,
 goal.AddEnergyBudget,Augmenter la production d'électricité,,
 goal.AddEnergyBudget.desc,"Nous sommes actuellement capables de produire {0}. La consommation de pointe a dépassé les 80%, donc pour éviter le déficit d'électricité, nous devons construire plus d'appareils capables de produire de l'électricité. Réacteurs à matière, panneaux solaires, tout ce pour quoi nous avons des plans. Si nous avons des panneaux solaires, les nettoyer pourrait aider.",,
 goal.BuildDiskModule,Construire un module de disque,,
-goal.BuildDiskModule.desc,Votre stockage de recherche s'épuise. Je suggère de construire un ou plusieurs modules de disque. Vous devrez faire des recherches en informatique pour acquérir les plans.,,
+goal.BuildDiskModule.desc,Votre <b>Stockage de recherche</b> s'épuise. Je suggère de construire un ou plusieurs <b>Modules de disque</b>. Vous devrez faire des recherches en informatique pour acquérir les plans.,,
```

Comment:
@LQ-Sebastien quick question, did you add more `<b> </b>` tags anywhere like in here? I didn't look for this in the last PR, but if you did, those translations will be broken. The game will see a mismatch in number of tags and will load English version instead. Please remove any extra HTML tags you've added.